tests/instructions/run-local.md

```diff
@@ -0,0 +1,87 @@
+
```

Comment:
The documents in directory "tests/instructions/" describe how some very specific development tasks can be performed, e.g. creating a "ninja trace", running cppcheck or sanitizers on the code. I actually don't like the fact that "tests/instructions/" exists. In an ideal world, all development instructions would be centrally available below the ["Contribute" part of the docs](https://clickhouse.com/docs/en/development) (which is build from "docs/en/development/...").

In this specific case, the content of this file could be inserted at the end of the "Functional Tests" section of the [testing guide](https://clickhouse.com/docs/en/development/tests).

Reply:
Agreed, I did not find the `tests/instructions` until recently.  I will have a look at it.